lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -217,6 +217,21 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
     // if organ donation campaign, add in the share link here
     if ($campaign->variables['register_organ_donors']) {
       $materials['content'] .= '<ul><li>Copy and paste your custom link to invite friends and family to become an organ donor: <code>' . $custom_organ_share_link . '</code></li></ul>';
+      drupal_add_js(
```

Comment:
not a huge deal, but another option is to store all the user stuff in a single object like. 

```
drupal_add_js([
 'dosomethingUser' => [
   'userInfo' => [
    'email' => $email,
     ...
   ]
 ]
]);
```
that way  in the js, you just need to set like 
`this.prefillvalues = setting('dosomethingUser.userInfo');`

 instead of having to grab them all individually and you can just loop through that object when you need the properties.